### PR TITLE
Repick data fix

### PR DESCRIPTION
**Pull request by LemurPwned**
Sort of solution example?
I dont think it is enough but it was a fast fix of #91

### CODE REVIEW

**Comment by pawelkulig1** [comment]
I will take a look.

**Comment by pawelkulig1** [comment]
This fix is breaking picking single files.

**Comment by LemurPwned** [comment]
Well I haven't changed the code for single files, since that was not the cause. But it might have been evoked by my changes anyway since the bug is really weird. It think it has smth to do with the focus policies but I'm not sure yet

**Comment by LemurPwned** [comment]
Ok now it should encompass everything.
It is stupid because we have to double click select dir/file but there is no other way around for now

**Comment by pawelkulig1** [comment]
Scenario:
Select directory (firstData) -> Matplotlib widget -> runAnimation -> select directory (Crash) - does not exists on repickData branch.

**Comment by LemurPwned** [comment]
Does it print something or it is sigkill?

**Comment by LemurPwned** [comment]
Ok, I seem not to be able to reproduce that bug with any of the datasets.
Are you certain you have latest?

File: main.py [hunk 1]
```diff
@@ -138,41 +143,49 @@ def setScreenshotFolder(self):
...
+            self.doh.passListObject(('odt_data', 'iterations'),
+                                        *MultiprocessingParse.readFile(fileLoaded))
             self._BLOCK_ITERABLES_ = False
             self._BLOCK_PLOT_ITERABLES_ = False
 
-        elif ".omf" in file or ".ovf" in filename:
-            self.doh.passListObject(('color_vectors', 'file_header'),
-                                    *MultiprocessingParse.readFile(filename))
+        elif ".omf" in fileLoaded or ".ovf" in fileLoaded:
+            self.doh.passListObject(('color_vectors', 'omf_header'),
```
Comment (LemurPwned) [inline main.py]: Here is the bug probably :)

**Pull request closed without merging**